crates/proof-of-sql/src/proof_primitive/dory/offset_to_bytes.rs

```diff
@@ -1,74 +1,57 @@
 use zerocopy::AsBytes;
 
 pub trait OffsetToBytes {
-    const IS_SIGNED: bool;
```

Comment:
The new version of clippy complained that this was unused.
@jacobtrombetta

Reply:
Thanks for taking care of the warning!